SPIFlashLogger.device.lib.nut

```diff
@@ -462,21 +549,26 @@ class SPIFlashLogger {
     }
 
     function _init() {
-        local best_id = 0;          // The highest id
-        local best_sec = 0;         // The sector with the highest id
-        local best_map = 0xFFFF;    // The map of the sector with the highest id
-
+        local first_sec = {"id" : 0, "sec" : 0, "map": 0xFFFF}; // The smallest id
```

Comment:
Can we please rename it to `firstSec` (or even `firstSector`) and the next one - to `lastSector`?

Reply:
replaced with firstSector/lastSector